titus-server-gateway/src/main/java/com/netflix/titus/gateway/kubernetes/KubeApiConnector.java

```diff
@@ -93,10 +93,12 @@ public KubeApiConnector(NamespacedKubernetesClient kubernetesClient, TitusRuntim
                 ContainerStatus containerStatus = iterator.next();
                 ContainerState status = containerStatus.getState();
                 TaskStatus.ContainerState.ContainerHealth containerHealth = TaskStatus.ContainerState.ContainerHealth.Unset;
-                if(status.toString().equals("running")) {
+                if(status.toString().equalsIgnoreCase("running")) {
```

Comment:
Can you take advantage of the client libs enums like `V1ContainerState.SERIALIZED_NAME_RUNNING` ?

Reply:
the fabric 8 kube client does not have that